lib/active_model_serializers/adapter/base.rb

```diff
@@ -51,6 +51,20 @@ def include_meta(json)
         json[meta_key] = meta if meta
         json
       end
+
+      def translate_key_casing!(value, serialization_context)
```

Comment:
I'm not necessarily opposed to it. Are there other benefits to this approach over what's in the PR?